### ISSUE TITLE
permission denied for creating pdf files

### ISSUE THREAD

**Issue by JafarMansouri** [jupyter/nbconvert#1524]
I want to create a pdf file using fpdf module. It worked before but recently it does work and gives the following error

```
import fpdf
pdf_tw = fpdf.FPDF(format='letter')
pdf_tw.add_page()
pdf_tw.set_font("Arial", 'B', size=18)
pdf_tw.write(5,'2000 first tweets that have been detected as relevant for business:')
pdf_tw.ln()
pdf_tw.ln()

pdf_tw.ln()
pdf_tw.output(name="tweets.pdf") 
```

PermissionError                           Traceback (most recent call last)
<ipython-input-25-f8556b305ab6> in <module>
      8 
      9 pdf_tw.ln()
---> 10 pdf_tw.output(name="tweets.pdf")

~\Anaconda3\lib\site-packages\fpdf\fpdf.py in output(self, name, dest)
   1077         elif dest=='F':
   1078             #Save to local file
-> 1079             f=open(name,'wb')
   1080             if(not f):
   1081                 self.error('Unable to create output file: '+name)

PermissionError: [Errno 13] Permission denied: 'tweets.pdf'

Thanks for any help.

**Comment by MSeal** [comment]
Are you running fpdf.py from a directory that you have write access to? What's your local directory? I'd debug the permissions / pathing first. Additionally this looks like a fpdf issue if it's a deeper problem rather than an nbconvert one.

**Comment by JafarMansouri** [comment]
Yes, I have access. I had written the files before with this code exactly on the same directory many times but suddenly this problem started. It is not a problem form pdpf since it occurs the other codes. Please see below. I understood that this problem happens just for the pdf and docx extensions. If I use txt extension this problem does not happen I run the program as the administrator. 

```
from pathlib import Path
p = Path('.') ## if you want to write to current directory
with open(p / 'testsss.pdf', 'w') as f:
    f.write('test message')
```

PermissionError                           Traceback (most recent call last)
<ipython-input-21-cb3237c347c1> in <module>
      1 from pathlib import Path
      2 p = Path('.') ## if you want to write to current directory
----> 3 with open(p / 'testsss.pdf', 'w') as f:
      4     f.write('test message')

PermissionError: [Errno 13] Permission denied: 'testsss.pdf
